plugins/openshift.yaml

```diff
@@ -0,0 +1,125 @@
+version: 0.0.1
+title: Openshift
+description: Log parser for Openshift
+parameters:
+  journal_directory:
+    label: Containers Log Path
+    description: The directory where the journald logs are located. Defaults to '/var/log/journal'.
+    type: string
+  start_at:
+    label: Start At
+    description: Start reading file from 'beginning' or 'end'. Defaults to 'end'.
+    type: string
+  container_logs:
+    label: Enable Container Logs
+    description: Enable collection of container logs
+    type: bool
+  docker_logs:
```

Comment:
Based on product team review of other plugins, use `docker_log` instead of `docker_logs`